app/models.py

```diff
@@ -17,13 +17,9 @@
 class User(AbstractUser):
     """
     Custom user model to use in the future.
-    Email is required to create an User.
     """
-
-    email = models.EmailField(null=False, unique=True, blank=False)
-    display_name = models.CharField(max_length=150, null=False, blank=True)
```

Comment:
`display_name` was used for case insensitive authentication, how does it work without it?

Reply:
two ways i was deleted:
one: because when you create superuser at first and login it you will get error forget argument
two: lower case insensitive dont work until when display_name found

Reply:
Ok, checked and case insensivity still works somehow.

Reply:
yes you're right about that, therefore i will re use it and now i will try to make project content more readable